molecule/default/molecule.yml

```diff
@@ -27,9 +27,7 @@ platforms:
     provider_options:
       cpu_mode: 'host-passthrough'
       nested: true
-    provider_raw_config_args:
-      - cputopology :sockets => '1', :cores => '4', :threads => '2'
-      - random :model => 'random'
+      machine_type: 'q35'
```

Comment:
im not sure why this is required but ok we can do this for now.
i would like toe keep the random number generator on eventually be the CPU topology is not working now so we can drop it